src/clusterfuzz/_internal/platforms/android/fetch_artifact.py

```diff
@@ -100,16 +100,25 @@ def get_artifacts_for_build(client, bid, target, attempt_id='latest'):
   request = client.buildartifact().list(
       buildId=bid, target=target, attemptId=attempt_id)
 
+  request_str = f'{request.uri}, {request.method},\
```

Comment:
Please use paren to seperate these strings instead of backslashes: https://google.github.io/styleguide/pyguide.html#32-line-length